config/initializers/active_record_extensions.rb

```diff
@@ -8,11 +8,11 @@ class ActiveRecord::Base
 # Permit safemode template rendering to have basic read-only access over
 # model relations
 class ActiveRecord::AssociationRelation::Jail < Safemode::Jail
-  allow :[], :each, :first, :to_a
+  allow :[], :each, :first, :to_a, :map
```

Comment:
We already have `#each`, I don't think `map` gives more power than each. It would be better to blacklist `#destroy`, since it's a dangerous action we don't want.